config/locales/it.json

```diff
@@ -19,7 +19,14 @@
             "remove": "Non sto piu' ignorando {user}!"
         },
         "lang": {
-            "title": "Lingue disponibili   💬"
+            "success": "Impostata lingua in: `{language}`!  {flag}",
```

Comment:
Thank you for updating this. Could you do this in a separate PR? I do not like to mix a whole bunch of different changes together.

But this is so, so appreciated!